### ISSUE TITLE
cannot migrate app

### ISSUE THREAD

**Issue by bryanwills** [kimlimjustin/password-manager#1]
When trying to run

`python manage.py migrate or python manage.py runserver
`

I get the following error:

`Traceback (most recent call last):
  File "/Users/bryanwills/code/github/password-manager/manage.py", line 22, in <module>
    main()
  File "/Users/bryanwills/code/github/password-manager/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 338, in execute
    django.setup()
  File "/usr/local/lib/python3.9/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.9/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python3.9/site-packages/django/apps/config.py", line 94, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/__init__.py", line 4, in <module>
    from django.contrib.admin.filters import (
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/filters.py", line 10, in <module>
    from django.contrib.admin.options import IncorrectLookupParameters
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/options.py", line 12, in <module>
    from django.contrib.admin import helpers, widgets
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/widgets.py", line 151
    '%s=%s' % (k, v) for k, v in params.items(),
    ^
SyntaxError: Generator expression must be parenthesized`

MacOS Big Sur, Intel Chip.

**Comment by kimlimjustin** [comment]
Wow, this project is almost a year which  I never touched after it's done, I'll look into it soon later

**Comment by kimlimjustin** [comment]
I think you should delete `db.sqlite3` file and run `python manage.py migrate` then `python manage.py runserver`, it works on Windows 11!

**Comment by bryanwills** [comment]
Wow that was a fast repsonse! I found your profile when I went down the Github wormhole the other night and I just now went back to it. So I tried this in a venv as well on my Mac and no luck. I cloned it on my WIndows 11 VM that I spun up since you say it works on Windows 11 and sure enough it worked. I had to delete the `db.sqlite3` file though. I thought it was more like a 1Password interface but these are really strong passwords and can work just as well. Doubt they will be cracked anytime soon!

Thanks again for the quick response! Not sure why it wouldn't work on my Mac though.

**Comment by kimlimjustin** [comment]
Haha, this project was the project I created when learning Django back then, looking at it now, yep, it got so many places to improve like interface and the strengths of the password (I know it's kind of ugly haha).

I don't think I'll make changes to this tho since I got another project ([xplorer](https://github.com/kimlimjustin/xplorer)), feel free to look into my other projects haha. 

Closing this for now, thanks!
